src/components/number-picker/number-picker.component.ts

```diff
@@ -192,9 +193,9 @@ export class NumberPickerComponent implements ControlValueAccessor, OnDestroy, O
         this.disabled = isDisabled;
     }
 
+    /** Set the value and emit the change to the output and Angular forms.
+    ** This is a workaround for angular bug https://github.com/angular/angular/issues/12540 */
```

Comment:
Could you remove this and make it a normal comment please? I had done this for you, but you overwrote it apparently.